ci/prow/make_config.go

```diff
@@ -765,6 +768,25 @@ func generateCleanupPeriodicJob() {
 	executeJobTemplate("periodic cleanup", periodicCustomJob, "presubmits", "", data.PeriodicJobName, false, data)
 }
 
+// generateFlakytoolPeriodicJob generates the cleanup job config.
+func generateFlakytoolPeriodicJob() {
+	var data periodicJobTemplateData
+	data.Base = newbaseProwJobTemplateData("knative/test-infra")
+	data.Base.Image = flakytoolDockerImage
+	data.PeriodicJobName = "ci-knative-flakytool"
```

Comment:
I always read "flakytool" as "a tool that is flaky". Can we use something more inline with the flaky-test-reporter name, but shorter, like "flaky-tests" or "flakes-reporter" or "flaky-checker"? (I like "flakes-reporter" the most)

Reply:
lol, I felt the same after reading your comment. Renamed to "flakes-reporter"